### PR TITLE
feat: balance added to subsidy serializer. get subsidy viewset perms …

### PR DESCRIPTION
**Pull request by iloveagent57**
…working right

### Description
* Get the permission objects and permission list stuff working right for `SubsidyViewSet`
* Add balance to subsidy serializer
* do some QOL stuff for local pytest running

https://2u-internal.atlassian.net/browse/ENT-6885

### Testing instructions

Add some, if applicable

### Merge checklist
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are squashed

### CODE REVIEW

File: enterprise_subsidy/settings/test.py [hunk 1]
```diff
@@ -13,4 +13,5 @@
         'PORT': '',
     },
 }
+DEBUG = False
```
Comment (pwnage101) [inline enterprise_subsidy/settings/test.py:16]: was this intentionally left in?

**Review by macdiesel — APPROVED** [review]

File: enterprise_subsidy/apps/api/v1/tests/mixins.py [hunk 2]
```diff
@@ -75,21 +75,23 @@ def setUp(self):
         self.desired_system_wide_role = None
         self.desired_feature_role = None
 
-    def set_up_admin(self):
+    def set_up_admin(self, enterprise_uuids=None):
```
Comment (pwnage101) [inline enterprise_subsidy/apps/api/v1/tests/mixins.py:78]: I don't object to adding this parameter, but for this PR it's a no-op right?  Just trying to make sure I understand what's happening.

**Review by pwnage101 — APPROVED** [review]
A couple minor things that looked odd, the rest LGTM!

File: enterprise_subsidy/settings/test.py (hunk 1 above)
Comment (iloveagent57) [reply enterprise_subsidy/settings/test.py:16]: no, that can come out

File: enterprise_subsidy/apps/api/v1/tests/mixins.py (hunk 2 above)
Comment (iloveagent57) [reply enterprise_subsidy/apps/api/v1/tests/mixins.py:78]: Yeah, I think I'll want this for more test cases that assert access isn't allowed.  Currently it's not doing anything (besides making it slightly more obvious that `set_up_admin()` calls are associating the test case user with that exact enterprise uuid)

**Pull request merged**